generators/app/index.js

```diff
@@ -139,7 +139,7 @@ module.exports = Generator.extend({
   install: {
     installDependencies: function() {
       var dependencies = [
-
+        'dotenv'
```

Comment:
I think this should be installed as a `dev` dependency since it's only being used to grab properties during build, right?